src/components/submit-project/SubmitProject.jsx

```diff
@@ -127,24 +72,6 @@ class SubmitProject extends Component {
               onChange={event => this.setState({ contact: event.target.value })}
             />
           </FormGroup>
-          <EditProjectImages handleUploadImages={this.handleUploadImages} />
```

Comment:
I can't upload files on submit project form? how can i test this feature?

Reply:
Currently, you should be able to test it by creating a new project, and then editing it. In a later commit, I removed `<EditProjectImages/>` from `SubmitProject.jsx` entirely, the idea being to put most of the editing features in `EditProject.jsx`. This is something I am still designing mockups for.

Reply:
<img width="1107" alt="Screen Shot 2019-04-08 at 10 09 30 PM" src="https://user-images.githubusercontent.com/12876643/55774680-0dad0280-5a4b-11e9-86b9-77593a595330.png">
First Error

Reply:
<img width="691" alt="Screen Shot 2019-04-08 at 10 11 48 PM" src="https://user-images.githubusercontent.com/12876643/55774761-67adc800-5a4b-11e9-9101-0e3c127a7e4b.png">
Refreshed the page, tried again, and got this.

Reply:
I also noticed that upon saving the changes the images do not show up on carousel until a page refresh?

Reply:
Ahh, I believe `ProjectPage` tries to save an entire project object to the database when the Save button is clicked. Unfortunately, this state is out of sync with the actual project images after you use the `EditProjectImages` picker. 

I fixed this in a later comment below by triggering a re-mount on save.